src/nodes.coffee

```diff
@@ -2512,6 +2538,12 @@ exports.Obj = class Obj extends Base
     answer = @wrapInBraces answer
     if @front then @wrapInParentheses answer else answer
 
+  checkForMultipleSplats: ->
+    return unless @hasSplat() and @lhs
+    props = @properties
+    splatProps = (i for prop, i in props when prop instanceof Splat)
```

Comment:
Can we not duplicate these lines? Maybe change `checkForMultipleSplats` to `splatProps` that both does the check and returns `splatProps`?

Reply:
Sure, updated to return `splatProps` and renamed to `getAndCheckSplatProps()`